test/sdk.test.ts

```diff
@@ -456,6 +456,29 @@ describe('Tests the SDK functionality', () => {
       expectNotIntegration('NativeFrames');
     });
 
+    it('no stall tracking integration by default', () => {
```

Comment:
NIT: By default it adds the integration, unless the tracing sample rate is not set.

```suggestion
    it('when tracing not set stall tracking integration not added by default', () => {
```

Reply:
Thanks, yeah I see it's a bit unclear.